bitcoin/src/psbt/error.rs

```diff
@@ -78,7 +79,7 @@ pub enum Error {
     /// Negative fee
     NegativeFee,
     /// Integer overflow in fee calculation
-    FeeOverflow,
+    FeeOverflow(amount::OutOfRangeError),
```

Comment:
I had just used `FeeOverflow` here.  Not a bad idea I suppose to be more specific, although does complicate the diff a bit.

Reply:
On further thought the `psbt` module only uses `Amount` (not signed) so we don't need this inner error. Removed.